### ISSUE TITLE
should cmi5 normative json-ld profile be in AICC repo?

### ISSUE THREAD

**Issue by phtyson** [AICC/CMI-5_Spec_Current#754]
I only find the cmi5 json-ld and RDFS files published at https://github.com/adlnet/xapi-authored-profiles/tree/master/cmi5. It seems the normative versions of these should be in AICC repository.

**Comment by phtyson** [comment]
After more research, I found that the profile loaded to http://profiles.usalearning.net/profiles is the original draft that appears at https://github.com/adlnet/xapi-authored-profiles/blob/master/cmi5/cmi5.jsonld and in the . The one at https://github.com/adlnet/xapi-authored-profiles/blob/master/cmi5/v1.0/cmi5.jsonld is more complete. It includes statement templates and patterns.
I don't know the provenance of the v1.0 cmi5 profile. Last commit was by Matthew (adl-trey) in Jan 2020. I looked at a few meeting minutes from that month but did not see any mention of it.
I suggest updating the cmi5 WG [working documents](https://github.com/AICC/CMI-5_Spec_Current/wiki/cmi5-%28JSON-LD%29--for-xAPI-Profile-%28DRAFT%29) to include latest published (v1) profile, and continue to review and improve it.
Also we should publish v1 to the current profile server.
